src/qz/printer/action/PrintRaw.java

```diff
@@ -188,7 +189,9 @@ private ImageWrapper getPdfWrapper(String data, JSONObject opt, boolean fromFile
         return getWrapper(bi, opt);
     }
 
-    private ImageWrapper getHtmlWrapper(String data, JSONObject opt, boolean fromFile, PrintOptions.Pixel pxlOpts) throws IOException {
+    private ImageWrapper getHtmlWrapper(String data, JSONObject opt, PrintingUtilities.Flavor flavor, PrintOptions.Pixel pxlOpts) throws IOException {
+        String source = flavor == PrintingUtilities.Flavor.BASE64 ?  new String(Base64.decodeBase64(data), StandardCharsets.UTF_8) : data;
```

Comment:
Probably the same concern as above - make this an if-statement instead

Reply:
Done.